token-syncer/src/token_syncer/commands/syncer.clj

```diff
@@ -97,6 +98,26 @@
                   {:code :error/token-read
                    :details {:message "token root missing from latest token description"}}
 
+                  (and latest-root
+                       (= latest-token-description description))
+                  {:code :success/token-match}
+
+                  ;; deleted on both clusters, irrespective of remaining values
+                  (and (get latest-token-description "deleted")
+                       (get description "deleted"))
+                  {:code :error/tokens-deleted
+                   :details {:message "token deleted on both clusters should be handled separately"}}
```

Comment:
What does "should be handled separately" mean?

Reply:
Updated the messaging.